commit 7f33628d9034af1345f97fde11fd6b8ec5acee1e (refs/changes/34/3334/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-08T15:04:00-08:00 (1 year, 8 months ago)
    
    volapi should only look for "ready" volumes when checking volumes reachability

diff --git a/lib/endpoints/vms.js b/lib/endpoints/vms.js
index 1c25c7e..33c09ec 100644
--- a/lib/endpoints/vms.js
+++ b/lib/endpoints/vms.js
@@ -1332,7 +1332,8 @@ function createVm(req, res, next) {
 
                 req.app.volapi.listVolumes({
                     name: volumeName,
-                    owner_uuid: ownerUuid
+                    owner_uuid: ownerUuid,
+                    state: 'ready'
                 }, function onListVols(listVolsErr, vols) {
                     var checkReachableErr;
                     var foundVolNetwork = false;
